Bottom: e5af899438bf61af290d32768e6d01c95e12a4fc
Top:    a590756ec8b824c6749c51d3f373ca6a82f872fc
Author: student <student@ReliableEmbeddedSystems.com>
Date:   2019-09-10 15:20:02 +0200

allow more recent docker version to be used together with docker-compose

Signed-off-by: student <student@ReliableEmbeddedSystems.com>


---

diff --git a/recipes-containers/docker-compose/files/0002-remove-docker-version-dependency.patch b/recipes-containers/docker-compose/files/0002-remove-docker-version-dependency.patch
new file mode 100644
index 0000000..a916e4b
--- /dev/null
+++ b/recipes-containers/docker-compose/files/0002-remove-docker-version-dependency.patch
@@ -0,0 +1,25 @@
+From 317fec6cbaa481428b69575639f3c7d6d0cf8325 Mon Sep 17 00:00:00 2001
+From: genius user <genius@ReliableEmbeddedSystems.com>
+Date: Tue, 16 Jul 2019 15:52:53 +0000
+Subject: [PATCH] remove docker version dependency
+
+Signed-off-by: genius user <genius@ReliableEmbeddedSystems.com>
+---
+ setup.py | 1 -
+ 1 file changed, 1 deletion(-)
+
+diff --git a/setup.py b/setup.py
+index f444757..dacbdb7 100644
+--- a/setup.py
++++ b/setup.py
+@@ -36,7 +36,6 @@ install_requires = [
+     'requests >= 2.6.1, != 2.11.0',
+     'texttable >= 0.9.0, < 0.10',
+     'websocket-client >= 0.32.0, < 1.0',
+-    'docker >= 2.5.1, < 3.0',
+     'dockerpty >= 0.4.1, < 0.5',
+     'six >= 1.3.0, < 2',
+     'jsonschema >= 2.5.1, < 3',
+-- 
+2.7.4
+
diff --git a/recipes-containers/docker-compose/python3-docker-compose_1.16.1.bb b/recipes-containers/docker-compose/python3-docker-compose_1.16.1.bb
index 851c251..e03c35f 100644
--- a/recipes-containers/docker-compose/python3-docker-compose_1.16.1.bb
+++ b/recipes-containers/docker-compose/python3-docker-compose_1.16.1.bb
@@ -3,7 +3,9 @@ HOMEPAGE = "https://www.docker.com/"
 LICENSE = "Apache-2.0"
 LIC_FILES_CHKSUM = "file://LICENSE;md5=435b266b3899aa8a959f17d41c56def8"
 
-SRC_URI += "file://0001-Allow-newer-versions-of-requests.patch"
+SRC_URI += "file://0001-Allow-newer-versions-of-requests.patch \
+            file://0002-remove-docker-version-dependency.patch \
+           "
 
 inherit pypi setuptools3
